File: event-service/src/main/java/com/github/manosbatsis/services/event/service/UserEventService.java
package com.github.manosbatsis.services.event.service;

import com.github.manosbatsis.services.event.model.UserEvent;

import java.util.List;

public interface UserEventService {

    List<UserEvent> findByUserId(Long id);

    List<UserEvent> findAll();

    UserEvent save(UserEvent userEvent);
}

File: event-service/src/main/kotlin/com/github/manosbatsis/services/event/service/UserEventService.kt
package com.github.manosbatsis.services.event.service

import com.github.manosbatsis.services.event.model.UserEvent

interface UserEventService {
    fun findByUserId(id: Long): List<UserEvent>
    fun findAll(): List<UserEvent>
    fun save(userEvent: UserEvent): UserEvent
}
